phoenix-driver/src/phx6_driver.cpp

```diff
@@ -648,59 +658,55 @@ Phoenix6Driver::~Phoenix6Driver()
 
 void Phoenix6Driver::getParams(ParamConfig& params)
 {
-    declare_param(
-        this,
-        "diagnostics_port",
-        params.diagnostics_server_port,
-        DIAG_SERVER_PORT);
-    declare_param<std::string>(this, "canbus", params.canbus, CAN_INTERFACE);
-    declare_param<std::string>(
-        this,
+    params.diagnostics_server_port =
```

Comment:
Why is this better? Seems like this would be worse for strings and equivalent for primitives.